pkg/core/localize/locales/en/cmd/kafka.en.toml

```diff
@@ -559,120 +559,120 @@ one = 'Format in which to display the Kafka topic (choose from: "json", "yml", "
 one = 'Format in which to display the Kafka topic (choose from: "json", "yml", "yaml")'
 
 [kafka.topic.produce.flag.key.description]
-one = 'The message key. Empty if not set'
+one = 'Message key. Empty if not set'
 
 [kafka.topic.produce.flag.key.file]
-one = 'Path to file containing message sent'
+one = 'Path to file containing sent message'
 
 [kafka.topic.produce.flag.partition.description]
-one = 'The partition number for the message. Must be positive integer value that represents number of partitions for the specified topic'
+one = 'Partition number for the message. Must be a positive integer value that is valid for the specified topic'
 
 [kafka.topic.produce.flag.format.description]
-one = 'Format of the messages printed as they are produced, possible values are json and yaml'
+one = 'Format for printing produced messages (possible values are "json" and "yaml")'
 
 [kafka.topic.produce.info.produceSuccess]
 description = 'Produce success message'
-one = 'Record successfully produced to "{{.Topic}}" with offset {{.Offset}}'
+one = 'Message successfully produced to "{{.Topic}}" with offset {{.Offset}}'
 
 [kafka.topic.produce.error.messageTooLarge]
-description = 'Error message when message produced was too large'
-one = 'The messaged sent was too large, the max size for a message is 1MB'
+description = 'Error message if produced message is too large'
+one = 'The sent message was too large, the maximum size for a message is 1MB'
 
 [kafka.topic.common.error.partitionNotFoundError]
-description = 'Error message when the partition given does not exist on the topic'
-one = 'The topic "{{.Topic}}" does not have the  partition {{.Partition}}'
+description = 'Error message when specified partition does not exist on the topic'
+one = 'Topic "{{.Topic}}" does not have a partition value {{.Partition}}'
 
 [kafka.topic.produce.input.value]
 description = 'Prompt for entering value to produce'
 one = 'Value'
 
 [kafka.topic.produce.input.help]
 description = 'Help message when entering value to produce'
-one = 'Enter text to be the value of the message produced'
+one = 'Enter text for message value to produce'
```

Comment:
It could just be me but I do not understand the new sentence here

Reply:
> It could just be me but I do not understand the new sentence here

Updated to "Enter text for the message value to be produced". Does that sound better, @jackdelahunt?

Reply:
Yes sounds good sorry the updated sentence was confusing me :smile: